modules/acs-architecture-differences-ocp-kube.adoc

```diff
@@ -7,10 +7,6 @@
 
 When you install {product-title-short} on the {ocp}, there are only two architectural differences:
 
-. {product-title-short} installs a lightweight version of Scanner on every secured cluster when you install {product-title-short} on the {ocp} using the Operator or the Helm install method.
-The lightweight Scanner enables the scanning of images in the integrated OpenShift Container Registry (OCR).
-. Sensor communicates with Scanner in the cluster where you have installed Central.
-This connection allows accessing internal registries attached to the cluster.
+. {product-title-short} installs a lightweight version of Scanner on every secured cluster when you install {product-title-short} on {ocp} by using the Operator. The lightweight Scanner enables the scanning of images in the integrated OpenShift Container Registry (OCR). When you install {product-title-short} by using the Helm install method with the _default_ values, the lightweight version of Scanner is not installed. To install the lightweight Scanner on the secured cluster by using Helm, you must set the `scanner.disable=false` parameter.
```

Comment:
Is `OpenShift Container Registry (OCR)` the official name for the registry? The [OCP docs](https://docs.openshift.com/container-platform/4.16/registry/index.html) are referring to it as the `OpenShift image registry`.

IMO it is more clear if referring to it as the `OCP Internal Registry` or `OCP Integrated Registry`.

Reply:
The Style Guide has "OpenShift Container Registry" - https://redhat-documentation.github.io/supplementary-style-guide/#openshift-container-registry

I can ask the OpenShift writers why there is a difference with the doc you linked to - I'm confused. I'm assuming that these are the same thing, though the internal, integrated registry that is created by OpenShift when it's installed...is that correct?

Reply:
I heard back from one of the writers and he is going to propose a change to our Style Guide since the newer usage is "integrated OpenShift image registry" as in the docs you linked. He discussed it with the Eng manager for the integrated registry and they determined that the SSG is just outdated.